dts/bindings/sensor/nxp,s32-qdec.yaml

```diff
@@ -0,0 +1,104 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Quadrature Decoder driver which processes encoder signals to determine motor revs
+
+  with the cooperation of S32 IP blocks- eMIOS, TRGMUX and LCU.
+
+  The sensor qdec application can be used for testing this driver.
+
+  The following example uses TRGMUX IN2 and IN3 to connect to LCU1 LC0 I0 and I1.
+
+  LCU1 LC0 O2 and O3 connect to eMIOS0 CH6(Clockwise rotation) and
+
+  CH7(Counter Clockwise rotation) via TRGMUX_INT_OUT37 and TRGMUX_INT_OUT38
+
+  micro-ticks-per-rev is set as per vehicle gearbox reduction.
+
+  lcu output filters are set to capture maximum speed sensitivity and avoid channel noise.
```

Comment:
This doesn't need to be double spaced. Remove the empty lines

Reply:
Removed empty lines !